components/common/autoTeleport/AutoTeleportActionButton.vue

```diff
@@ -0,0 +1,277 @@
+<template>
+  <div class="is-flex is-flex-direction-column w-full">
+    <div
+      v-if="showAutoTeleport"
+      class="is-flex is-justify-content-space-between w-full mb-4">
+      <div class="is-flex">
+        <div class="has-accent-blur">
+          <img :src="autoTeleportIcon" class="mr-2" alt="teleport arrow" />
+          <img
+            v-if="isTelportIconActive"
+            src="/accent-blur.svg"
+            alt="blur"
+            class="blur autotelport-blur" />
+        </div>
+
+        <p
+          class="has-text-weight-bold"
+          :class="{ 'has-text-k-grey': !hasAvailableTeleportTransition }">
+          {{ $t('autoTeleport.autoTeleport') }}
+        </p>
+
+        <AutoTeleportTooltip
+          v-if="hasAvailableTeleportTransition"
+          position="top"
+          :transition="optimalTransition" />
+      </div>
+
+      <div
+        v-if="!hasAvailableTeleportTransition"
+        class="is-flex is-align-items-center"
+        :class="{ 'has-text-k-grey': !hasAvailableTeleportTransition }">
+        <span class="is-size-7">{{ $t('autoTeleport.notAvailable') }}</span>
+
+        <AutoTeleportTooltip position="left" :transition="optimalTransition" />
+      </div>
+
+      <NeoSwitch
+        v-else
+        v-model="autoTeleport"
+        data-testid="auto-teleport-switch" />
+    </div>
+
+    <NeoButton
+      :label="autoTeleportLabel"
+      variant="k-accent"
+      no-shadow
+      :disabled="isDisabled"
+      class="is-flex is-flex-grow-1 btn-height is-capitalized"
+      @click="submit" />
+
+    <div v-if="showAutoTeleport" class="is-flex is-justify-content-center mt-4">
+      <span v-if="hasAvailableTeleportTransition" class="has-text-grey"
+        >Or</span
+      >
+
+      <a class="ml-2" @click="onRampActive = true"
```

Comment:
I would change it with a text button (to avoid undefined href)

Reply:
fixed